packages/rum-core/src/domain/contexts/pageStateHistory.ts

```diff
@@ -98,6 +99,10 @@ export function startPageStateHistory(
 
       return pageStateServerEntries
     },
+    isActiveAt: (startTime: RelativeTime) => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏more explicit naming
```suggestion
    isInActivePageStateAt: (startTime: RelativeTime) => {
```